Mathlib/MeasureTheory/Function/AEEqFun.lean

```diff
@@ -849,6 +858,21 @@ theorem coeFn_abs {β} [TopologicalSpace β] [Lattice β] [TopologicalLattice β
 
 end Abs
 
+section Star
+
+variable {R : Type*} [TopologicalSpace R]
+
+instance [Star R] [ContinuousStar R] : Star (α →ₘ[μ] R) where
+  star f := (AEEqFun.comp _ continuous_star f)
+
+lemma coeFn_star [Star R] [ContinuousStar R] (f : α →ₘ[μ] R) : ↑(star f) =ᵐ[μ] (star f : α → R) :=
+  coeFn_comp _ (continuous_star) f
+
+instance [InvolutiveStar R] [ContinuousStar R] : InvolutiveStar (α →ₘ[μ] R) where
```

Comment:
I assume your plan is to prove `StarMul` etc in a later PR?

Reply:
Exactly. Let me try to add `StarTrivial` here.

Reply:
I included a proof of that, but I'm sure it can be golfed. The behavior of star_trivial is a little different than star_involutive...so the term mode is fighting back...